src/algorithms/chain_items.hpp

```diff
@@ -479,6 +577,11 @@ int score_best_chain(const VectorView<Anchor>& to_chain, const SnarlDistanceInde
 /// This produces a penalty (positive number).
 int score_chain_gap(size_t distance_difference, size_t average_anchor_length);
 
+/// Determine if adding the new anchor would cause a recombination event
+/// with respect to the old anchor, given their supported paths.
+/// Returns 0 if no recombination, or 1 if there is a recombination.
```

Comment:
It's a bit weird that this is an `int`; I guess it's so you can do math with it without doing weird stuff like multiplying by `bool`?